src/utils/agent.ts

```diff
@@ -102,7 +102,12 @@ const Projects = {
 }
 
 const Opportunities = {
-    list: () => requests.get<Opportunity[]>('opportunities'),
+    list: (params?: URLSearchParams) => requests.get<Opportunity[]>('opportunities', params),
+    post: (body: Opportunity) => requests.post<Opportunity>('opportunities', body),
+    getCount: () => requests.get<Object>('opportunities/count'),
+    getDetails: (id: string) => requests.get<Opportunity>(`opportunities/${id}`),
+    update: (id: string, body: Opportunity) => requests.put<Opportunity>(`opportunities/${id}`, body),
+    delete: (id: string) => requests.delete<Object>(`opportunities/${id}`),
```

Comment:
id should be number

Reply:
Hello @Enjoy2Live Thanks for the review. Had some confusions cause in api doc i got ->
![Screenshot 2022-06-27 at 10 32 49 PM](https://user-images.githubusercontent.com/15964566/175995934-e11b1899-f495-47d6-9245-5f5cdf1959c9.png)
id as string

![Screenshot 2022-06-27 at 10 33 50 PM](https://user-images.githubusercontent.com/15964566/175996006-86c1da89-cdb8-4634-8256-083071fbad48.png)
and getCount api is returning object.

Should i still proceed?

Reply:
it's a number

Reply:
also the id is technically a number but it's not a huge deal if you keep it string